lib/manager/gitlabci/extract.spec.ts

```diff
@@ -1,34 +1,61 @@
-import { readFileSync } from 'fs';
-import { extractPackageFile } from './extract';
-
-const yamlFile = readFileSync(
-  'lib/manager/gitlabci/__fixtures__/gitlab-ci.yaml',
-  'utf8'
-);
-
-const yamlFile1 = readFileSync(
-  'lib/manager/gitlabci/__fixtures__/gitlab-ci.1.yaml',
-  'utf8'
-);
+import { PackageDependency } from '../common';
+import { extractAllPackageFiles } from './extract';
 
 describe('lib/manager/gitlabci/extract', () => {
```

Comment:
```suggestion
describe(getName(__filename), () => {
```
import `getName` from `test/utils`